src/ontology/mondo-edit.obo

```diff
@@ -302347,7 +302347,7 @@ consider: UMLS:C0014175 {source="Orphanet:137820"}
 [Term]
 id: MONDO:0015306
 name: Lemierre syndrome
-def: "and that is characterized by septic thrombophlebitis of the internal jugular vein that leads to septic, usually pulmonary, embolism, associated with ENT (ear, nose, and throat) infection that manifests with fever, neck pain, and tonsillopharyngitis." [Orphanet:137839]
+def: "And that is characterized by septic thrombophlebitis of the internal jugular vein that leads to septic, usually pulmonary, embolism, associated with ENT (ear, nose, and throat) infection that manifests with fever, neck pain, and tonsillopharyngitis." [Orphanet:137839]
```

Comment:
this one seems like it is truncated

Reply:
The def should be:

Lemierre syndrome is a rare, potentially lethal, oropharyngeal infectious disease occurring in immunocompetent adolescents and young adults that is mainly due to Fusobacterium necrophorum and that is characterized by septic thrombophlebitis of the internal jugular vein that leads to septic, usually pulmonary, embolism, associated with ENT (ear, nose, and throat) infection that manifests with fever, neck pain, and tonsillopharyngitis.